team.html

```diff
@@ -121,82 +109,90 @@ <h3>Controls</h3>
                     <div class="col-md-2 profile">
                         <img src="images/team/Tammy_Liu.png"/>
                         <h4 class="info">Tammy Liu</h4>
+                        <small>Advisor</small>
+                    </div>   
+                    <div class="col-md-2 profile">
+                        <img src="images/team/Jeff_Niu.png"/>
+                        <h4 class="info">Jeff Niu</h4>
                         <small>Team Lead</small>
-                    </div>                    
+                    </div>
                     <div class="col-md-2 profile">
                         <img src="images/team/Sadman_Kazi.jpg"/>
                         <h4 class="info">Sadman Kazi</h4>
-                        <small>Technical Lead</small>
```

Comment:
Sadman is also an Advisor now. Sorry if the team registry doc wasn't updated properly :\